spec/lib/efile/md/md502_calculator_spec.rb

```diff
@@ -1263,15 +1306,36 @@
   end
 
   describe "#calculate_line_28_local_tax_amount" do
+    let(:income) { 300_000 }
     before do
-      allow_any_instance_of(described_class).to receive(:calculate_line_20).and_return 300_000
+      allow_any_instance_of(described_class).to receive(:calculate_line_20).and_return income
       instance.calculate
     end
 
     context "when county is Anne Arundel" do
       let(:county) { "Anne Arundel" }
-      it "calculates the local tax" do
-        expect(instance.lines[:MD502_LINE_28_LOCAL_TAX_AMOUNT].value).to eq(8375)
+
+      context "when income is 300,000" do
+        # (0.027 * 50,000) + (0.0281 * 250,000) = 8375
+        it "calculates the local tax progressively" do
+          expect(instance.lines[:MD502_LINE_28_LOCAL_TAX_AMOUNT].value).to eq(8375)
+        end
+      end
+
+      context "when income is 450,000" do
+        let(:income) { 450_000 }
+        it "calculates the local tax progressively" do
+          # (0.027 * 50,000) + (0.0281 * 350,000) + (.032 * 50,000) = 12785
+          expect(instance.lines[:MD502_LINE_28_LOCAL_TAX_AMOUNT].value).to eq(127_85)
```

Comment:
[Dust] i'm not really certain why we use underscores for integers 😓 I'm guessing for readability? but should this be `12_785`?

Reply:
@arinchoi03 yes it's just readability! Its equal to a comma, so you're totally right it should be `12_785`